### ISSUE TITLE
VCF_OVERLAP undeclared

### ISSUE THREAD

**Issue by terrycojones** [samtools/bcftools#1177]
I'm trying to build from the latest sources. Following the `INSTALL` file, I ran `autoreconf` then `./configure` then `make`.  A bunch of files get compiled cleanly, but then:

```sh
$ make
gcc -Wall -g -O2 -I. -I../htslib   -c -o vcffilter.o vcffilter.c
vcffilter.c: In function 'main_vcffilter':
vcffilter.c:477:90: error: 'VCF_OVERLAP' undeclared (first use in this function)
                         else if ( !strcasecmp(keys[i],"overlap") ) args->snp_gap_type |= VCF_OVERLAP;
                                                                                          ^
vcffilter.c:477:90: note: each undeclared identifier is reported only once for each function it appears in
make: *** [Makefile:136: vcffilter.o] Error 1
```

A `make filter.o` or `make convert.o` gets the same error. I can't see `VCF_OVERLAP` defined anywhere, it's just used in those 3 files. The closest candidate (by name at least) is `VCFBUF_OVERLAP_WIN`.

**Comment by valeriuo** [comment]
Duplicate of #1166
